src/build.py

```diff
@@ -726,6 +726,10 @@ def get_rev(rev_name):
 
 
 def Clobber():
+  # Don't ever clobber non-bot (local) work directories
+  if not buildbot.IsBot():
```

Comment:
Should this just be rolled into `ShouldClobber()`?

Reply:
Or do you mean this whole function?  If thats what you meant I'm not sure it really matters.

Reply:
Oh I see what you mean. buildbot.ShouldClobber just checks the buildbot-controlled clobbering. This change is fine; separately we should just remove ShouldClobber since we don't have buildbot anymore.